src/main/java/org/openstreetmap/atlas/geography/geojson/GeoJsonBuilder.java

```diff
@@ -231,6 +233,36 @@ public GeoJsonObject create(final PolyLine polyLine)
         return this.create(polyLine, GeoJsonType.LINESTRING);
     }
 
+    /**
+     * Creates a GeoJson FeatureCollection containing a list of GeoJsonObject Features
+     *
+     * @param objects
+     *            the features
+     * @return a GeoJson FeatureCollection
+     */
+    public GeoJsonObject createFeatureCollection(final Iterable<GeoJsonObject> objects)
+    {
+        final JsonObject result = new JsonObject();
+        result.addProperty(TYPE, FEATURE_COLLECTION);
+        final JsonArray features = new JsonArray();
+        int counter = 0;
+        for (final GeoJsonObject object : objects)
+        {
+            if (this.logFrequency > 0 && ++counter % this.logFrequency == 0)
```

Comment:
This function mimics another in this class that I did not write. I am not 100% sure of the utility but I am matching the functionality. This will all be cleaned up in an upcoming PR with improvements to code surrounding Geojson